lib/studentscreen.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+
+class StudentScreen extends StatelessWidget {

```

Comment:
Wouldn't this be what the PrepHQ guys were referring to as the "My Mentor" section? Or were you going for like, the start of the basic profile page where the student could edit their info?